src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts

```diff
@@ -185,10 +201,14 @@ export class ExecutionPlanComparisonEditorView {
 		this._topPlanDropdown = new SelectBox(['option 1', 'option2'], 'option1', this.contextViewService, this._topPlanDropdownContainer);
 		this._topPlanDropdown.render(this._topPlanDropdownContainer);
 		this._topPlanDropdown.onDidSelect(async (e) => {
-			if (this._activeBottomPlanDiagram) {
-				this._activeBottomPlanDiagram.clearSubtreePolygon();
+			if (this.activeBottomPlanDiagram) {
```

Comment:
FYI - you can simplify these kind of calls by using optional chaining `this.activeBottomPlanDiagram?.clearSubtreePolygon()`